google/ggrc-core
# GGRC-2474 Migrate existing person global custom attributes into custom roles

# Dependencies

This PR is `on hold` until the dependencies are merged:
- [x] #7001 (FE, should be merged together)
- [x] #6891 (may break Revert Revision logic for Global CA of type Person)
- [x] #6817 (migration for the release branch)

# Issue description
1. Remove possibility to add global custom attributes with Person type for any object.
2. Migrate existing person global custom attributes into custom roles that by default will have no rights.

# Steps to test the changes
1. Try to add global custom attributes with Person type for any object. -> 'Invalid attribute' error message.
2. Run migration and make sure that all GCA with Person type are migrated to Custom Roles.

# Solution description
I have created mapping from 'custom_attribute_definitions' table to 'access_control_roles' table.
**Audits**, **People**, **Risk Assessments** and **Workflows** definition types are not migrated because we have no corresponding **Custom Roles**.

# Sanity checklist


# Migration checklist

# PR Review checklist
